### PR TITLE
Move pipeline operator to Stage 2

### PR DESCRIPTION
**Pull request by sdegutis**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13754"></a>

### CODE REVIEW

File: packages/babel-standalone/src/preset-stage-1.ts [hunk 1]
```diff
@@ -7,8 +7,6 @@ export default (_: any, opts: any = {}) => {
     useBuiltIns = false,
     decoratorsLegacy = false,
     decoratorsBeforeExport,
-    pipelineProposal = "minimal",
-    pipelineTopicToken = "%",
     recordAndTupleSyntax,
```
Comment (nicolo-ribaudo) [inline packages/babel-standalone/src/preset-stage-1.ts:10]: We should keep these options here (without the default values), and forward them to the Stage 2 preset at lines 18-22.

File: packages/babel-standalone/src/preset-stage-2.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ export default (_: any, opts: any = {}) => {
     useBuiltIns = false,
     decoratorsLegacy = false,
     decoratorsBeforeExport,
+    pipelineProposal = "hack",
```
Comment (nicolo-ribaudo) [inline packages/babel-standalone/src/preset-stage-2.ts:10]: Unfortunately changing the default is a breaking change :sweat_smile: We cannot default to `hack` until Babel 8.

File: packages/babel-standalone/src/preset-stage-1.ts (hunk 1 above)
Comment (sdegutis) [reply packages/babel-standalone/src/preset-stage-1.ts:10]: Ok, will update these 2 things later tonight. Tried to do it all inside GH's web editor and it's too difficult. But gotta get back to work now.

File: packages/babel-standalone/src/preset-stage-2.ts (hunk 2 above)
Comment (sdegutis) [reply packages/babel-standalone/src/preset-stage-2.ts:10]: Isn't that going to be really soon though?

Comment (nicolo-ribaudo) [reply packages/babel-standalone/src/preset-stage-2.ts:10]: Soon as in "a few months, maybe end 2021"; this PR should be a quick merge (either next patch or next minor, I'm not sure if we should consider it as a bugfix or as a new feature for the stage 2 preset).

Comment (sdegutis) [reply packages/babel-standalone/src/preset-stage-2.ts:10]: Ok will revert it later tonight when I update PR.

File: packages/babel-standalone/src/preset-stage-1.ts (hunk 1 above)
Comment (sdegutis) [reply packages/babel-standalone/src/preset-stage-1.ts:10]: I'm confused, do you mean the options should stay here and be imported by stage 2? And that the plugin lines below should remain here but not pass options, and be copied into stage 2 with the imported options? That's what it sounds like you're saying but it doesn't make a lot of sense to me to have no options for it here.

Comment (nicolo-ribaudo) [reply packages/babel-standalone/src/preset-stage-1.ts:10]: It should do the same as it does, for example, for `decoratorsBeforeExport`: it accepts the option (without setting a default value), and passes it as-is to the Stage 2 preset below (which then sets the default value).

**Comment by sdegutis** [comment]
@nicolo-ribaudo just updated it with the changes you requested, I think it now does what you want (had to do a -f push).

**Review by fedeci — APPROVED** [review]

**Comment by js-choi** [comment]
Ah, I had forgotten about the Babel presets. Thanks for doing this work.

**Review by js-choi — APPROVED** [review]

**Comment by sdegutis** [comment]
@js-choi @tabatkins if you guys want any more help moving forward pipelines in practice just lmk. I also plan to do the VS Code syntax highlighter change this evening (assuming they haven't), and hoping someone will update the TypeScript type checker soon to at least enable it under a stage2 flag, so I can start using it in code this year maybe.

**Review by nicolo-ribaudo — APPROVED** [review]
Thanks!

**Comment by sdegutis** [comment]
@nicolo-ribaudo Thanks for helping me figure out how to write this PR. Let me know if I can be any more help with pipelines.

**Pull request merged**